### PR TITLE
add armstrong code

### PR DESCRIPTION
**Pull request by akshatjain24**
# Pull Request Template

**Feature Name: Armstrong number**

**Changes made:**<br>
Added the code for Armstrong number

### Self Check(Tick After Making pull Request)

- [x] I have read and followed the [Contribution Guidelines](/CONTRIBUTING.md)
- [x] This issue was assigned to me.
- [x] One Change in one Pull Request
- [x] My file is in the proper folder

- [x] PR is regarding Code
- [x] I am following clean code and Documentation and my code is well linted with flake8.
- [ ] I am using external libraries and I have created requirements.txt in my project folder
- [x] Script is tested and running perfectly fine on my system

- [ ] PR is regarding the documentation

> If the issue was not assigned to you Please don't make a PR, It will be marked as invalid.

Discuss in discord server:- http://pragmaticprogrammer.in/discord

## acknowledgment
- [x] I have tested my code and distributing it under GPL-3.0 License

### CODE REVIEW

File: Math/Armstrong.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+x = input()
+sum = 0
+for i in x:
+    z = int(i)
+    sum += z**3
```
Comment (Punit-Choudhary) [inline Math/Armstrong.py:5]: I believe Armstrong numbers are the numbers whose sum of each integer is raised to the power length of the number is the same as the number.
eg: 
- 370 = 3^3 + 7^3 + 0^3
- 1634 = 1^4 + 6^4 + 3^4 + 4^4

**Review by Punit-Choudhary — CHANGES_REQUESTED** [review]

File: Math/Armstrong.py (hunk 1 above)
Comment (akshatjain24) [reply Math/Armstrong.py]: yes, you are right. Sorry, for the inconvenience. I have done the changes.

**Comment by akshatjain24** [comment]
@Punit-Choudhary I have updated the code, Please merge it so I have valid PR for my first hactoberfest.

Thanks for your support

**Review by Punit-Choudhary — APPROVED** [review]

**Comment by Punit-Choudhary** [comment]
@akshatjain24
Thank you for your contribution 💜 🚀

**Pull request merged**